apps/api/src/modules/users/dto/create-user.input.ts

```diff
@@ -0,0 +1,10 @@
+import { InputType, Field } from '@nestjs/graphql';
+
+@InputType()
+export class CreateUserInput {
```

Comment:
We don't need this for the time being. Can be added later when add the functionality to create new users.

Reply:
Ok, will remove